docs/guides/mozilla-sops.md

```diff
@@ -131,10 +131,24 @@ AWS IAM Role example:
 }
 ```
 
+Azure:
+
 When using Azure Key Vault you need to authenticate the kustomize controller either by passing
 [Service Principal credentials as environment variables](https://github.com/mozilla/sops#encrypting-using-azure-key-vault)
 or with [add-pod-identity](https://github.com/Azure/aad-pod-identity).
 
+GCP Role:
```

Comment:
```suggestion
Goggle Cloud:
```

Reply:
I would make all these `####` headers so that they show up in the TOC, and change the `
### AWS/Azure/GCP` to something that describes cloud provider specific settings.